### PR TITLE
Add <meta> tag for better look of html on mobile phones

### PR DESCRIPTION
**Pull request by horakmar**

### CODE REVIEW

File: quickhttpd/src/application.h [hunk 1]
```diff
@@ -34,6 +34,8 @@ class Application : public QCoreApplication
 	static Application *instance();
 	AppCliOptions* cliOptions() {return m_cliOptions;}
 	qf::core::sql::Connection sqlConnetion();
+
+  QString profile();
```
Comment (fvacek) [inline quickhttpd/src/application.h:38]: odsazeni

**Review by fvacek — CHANGES_REQUESTED** [review]
What is the use-case for this change?

File: quickhttpd/src/application.h (hunk 1 above)
Comment (horakmar) [reply quickhttpd/src/application.h:38]: Sorry, only first commit should go to pull request. I messed it up, last two were my experiments, I intended to try them in my fork only and forget to make separate branch for them.
For your information:
We plan to use quickevent, quickshow and quickhttpd at Sudety OPEN competition, held in Ostrov u Tise. There is no internet connection and we want to make Wifi island (in Ostrov :-)) for publishing of results. Competitors will check their results mainly on their mobile phones.

Comment (horakmar) [reply quickhttpd/src/application.h]: Indentation fixed, but all adding --profile option to quickhttpd is only my experiment. I have very little knowledge of C++ and did it only blindly following the pattern in quickshow.
I just need to render results, based on database. I do not need startlists, I even do not need http server, I'll use nginx. Luckily I can avoid creating http server using option --http-port 0 (not mentioned in help).

Comment (horakmar) [reply quickhttpd/src/application.h]: Still one issue remains: How to tell quickhttpd that I want to render pages in particular language (Czech)? It seems, that ResultExport in quickevent it does right using language setting of quickevent, but how to do it in quickhttpd?

Comment (otahirs) [reply quickhttpd/src/application.h]: > Sorry, only first commit should go to pull request. I messed it up, last two were my experiments, I intended to try them in my fork only and forget to make separate branch for them.

In order to merge the pull request, please revert any non-related commits.
run `git revert {commit hash}` for each, e.g. `git revert 9b3c8fd` will revert `Fix indent` commit,
or reset the branch to include only first commit:
```
// make sure you are on html_responsive branch
git reset --hard d51cee1
git push --force origin html_responsve
```

Comment (horakmar) [reply quickhttpd/src/application.h]: OK, done.

Comment (fvacek) [reply quickhttpd/src/application.h]: > I just need to render results, based on database. I do not need startlists

Can you just silently ignore the start-lists generated?

Comment (horakmar) [reply quickhttpd/src/application.h]: Of course I do. If it has no/minimal impact on performance.
It was just an experiment, how hard is C++ programming for me. :-) (Quite a lot.)

Comment (fvacek) [reply quickhttpd/src/application.h]: try to start with #625 :)

**Comment by paukert** [comment]
Wasn't this PR closed by mistake?

**Comment by horakmar** [comment]
No, it's OK for me.

**Comment by horakmar** [comment]
But you could merge that simple <meta> tag. Generated pages look much better on mobile devices then.
I wrote my own HTML export anyway (using Python and Jinja2).

**Comment by fvacek** [comment]
Yes it was :)

**Pull request merged**